kolibri/plugins/management/assets/src/device_management/views/welcome-modal.vue

```diff
@@ -42,6 +43,9 @@
       welcomeButtonDismissText: 'OK',
     },
     components: { coreModal, kButton },
+    created: function() {
+      window.addEventListener('keyup', this.emitCloseModal);
+    },
```

Comment:
This event listener will cause _any_ keypress to close the window, which is not desirable behavior.